tools/krona/tool_dependencies.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="True" />
+    </package>
+
```

Comment:
:)
You need to include something like:
```
<package name="krona" version="2.5">
        <repository name="package_krona_2_5" owner="iuc" />
</package>
```
The perl requirement can go away, imho.
To stop the confusion commit your tools to IUC as well :)